src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax;
+
+import core.basesyntax.exeptions.RegisterServiceException;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private RegistrationServiceImpl registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_invalidUser_NotOk() {
+        Assertions.assertThrows(RegisterServiceException.class, () ->
+                registrationService.register(null));
+        Assertions.assertThrows(RegisterServiceException.class, () ->
+                registrationService.register(new User(null, null, null, null)));
+        User invalidUser = new User(1L, "Login", "Hello!", 18);
+        Assertions.assertThrows(RegisterServiceException.class, () ->
+                registrationService.register(invalidUser));
```

Comment:
create separate test for each negative case, like invalidPass, invalidLogin, invalidAge